scripts/workers/code_upload_worker_utils/monitor_submission.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+echo "Monitoring submission..."
+time_elapsed=0
+while [ $time_elapsed -le $SUBMISSION_TIME_LIMIT ]
+do
+    echo "Checking for submission file after $time_elapsed secs.."
+    sh /evalai_scripts/make_submission.sh
+    if [ -f "$SUBMISSION_PATH/completed.txt" ]
+    then
+        echo "Submission Complete, Exiting."
+        exit 0
+    fi
+    if [ $time_elapsed -lt $SUBMISSION_TIME_LIMIT -a $(( $time_elapsed + $SUBMISSION_TIME_DELTA )) -gt $SUBMISSION_TIME_LIMIT ]
+    then
+        next_checkpoint=$SUBMISSION_TIME_LIMIT
```

Comment:
@savish28 please change the variable name to something like `time_delta` or `wait_interval`

Reply:
@Ram81 do you mean changing `SUBMISSION_TIME_DELTA` to `time_delta`? `next_checkpoint` would be the next time at which we would be going to check.

Reply:
I meant change the name of `next_checkpoint` to something relevant to next time interval or similar

Reply:
Updated.